packages/lib/contracts/Initializable.sol

```diff
@@ -32,12 +32,18 @@ contract Initializable {
     require(initializing || isConstructor() || !initialized, "Contract instance has already been initialized");
 
     bool wasInitializing = initializing;
-    initializing = true;
-    initialized = true;
+    if (!wasInitializing) {
```

Comment:
```suggestion
    if (!initializing) {
```

This is more straightforward, less confusing, and makes it clear that this `if` block and the next one are doing exactly the same thing.